test/integration/extensionRequests.test.js

```diff
@@ -40,13 +41,15 @@ describe("Extension Requests", function () {
 
   before(async function () {
     const userId = await addUser(user);
+    const userId2 = await addUser(user2);
```

Comment:
```suggestion
    const userId2 = await addUser(userData[5]);
```

no need for variable if it is not being used in multiple places or required